tests/website.py

```diff
@@ -16,6 +16,9 @@ def _pre_setup_10_website(self):
         self.deployment.add("haproxy")
         self.deployment.relate("jenkins:website", "haproxy:reverseproxy")
         self.deployment.expose("haproxy")
+        self.deployment.add("nrpe")
+        self.deployment.relate("jenkins:nrpe-external-master",
+                               "nrpe:nrpe-external-master")
```

Comment:
It would be better to add a separate test file for this relation. The reason being that if there are problems in the haproxy relation and, say, the the deployment fails, then nrpe tests will appear to be broken as well, even if in fact they are not (i.e. the issue is only in the haproxy relation).